ble/plf/hci_uart.c

```diff
@@ -21,7 +21,7 @@
 LOG_MODULE_REGISTER(hci_uart, CONFIG_UART_LOG_LEVEL);
 
 /* Define appropriate timeouts */
-#define TX_TIMEOUT_US 10 /* 10us */
+#define TX_TIMEOUT_US 200 /* 200us */
```

Comment:
What apppropriate actually means in this case?

Reply:
timeout for dma tx? do you want add bigger?